site/web/app/themes/sage/assets/styles/components/resource-card.sass

```diff
@@ -1,7 +1,7 @@
 .resource-card
-    float: left
     width: 45%
-    height: 200px
+    height: auto
+    flex: 0 1 auto
     margin: 22.5px 2.5%
```

Comment:
you can remove the left and right margin and keep the 22.5px top and bottom margin